### ISSUE TITLE
Compile error using nightly compiler

### ISSUE THREAD

**Issue by pgerber** [netvl/xml-rs#148]
Looks like xml-rs stopped compiling with the nightly compiler from 2017-05-21. Nightly build from 2017-05-20 still worked.

```
error: no rules expected the token `flags`
   --> /home/pg/.cargo/registry/src/github.com-1ecc6299db9ec823/xml-rs-0.1.26/src/writer/emitter.rs:115:5
    |
115 |     flags IndentFlags: u8 {
    |     ^^^^^

error: Could not compile `xml-rs`.
```

**Comment by pgerber** [comment]
Never mind, assumed it was an error in xml-rs because it still compiled with yesterdays nightly compiler but it looks like the release of bitflags 0.9.0 caused the issue.

**Comment by jens1o** [comment]
I hit this today. Does anyone know how to circumvent this? Seems that this project is no longer maintained. :/

**Comment by netvl** [comment]
Dependency on bitflags got removed in the reimplementation branch. I guess I can add a fix to the main branch too.

**Comment by netvl** [comment]
@jens1o that being said, I'm not sure how do you see this issue. Right now, the library does not already depend on bitflags (forgot about it!). Also, I've tried installing the current nightly and running tests with it, and it worked fine. Could you please provide more details?

**Comment by jens1o** [comment]
Thanks for your reply and maintenance of this project! I was using the beta compiler yesterday, and I think I tried to add a piston related dependency I don't really know which one though (iirc, something with 2d). :( The error message was identical to the one posted in first comment in 2017. However, I was able to somehow achieve the very same result with a different piston dependency and worked-around the issue.

I'm sorry I can't give any more detailed steps how to reproduce.
